client/src/pages/Landing/AuthNav.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import { Container, Button } from '@material-ui/core';
+import ForumRoundedIcon from '@material-ui/icons/ForumRounded';
+
+import image from '../../images/woman-laptop.png';
```

Comment:
We should use absolute imports
```suggestion
import image from 'images/woman-laptop.png';
```